assignment.js

```diff
@@ -64,6 +69,14 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let numbers = [1,2,3]
+console.log(`Numbers are ${numbers}`)
+let sum = 0
+numbers.forEach(function(element,index){
```

Comment:
@kertcodes going forward, you don't need to include the `index` parameter for `.forEach()` if you're not going to use it in the callback function